lib/puma/reactor.rb

```diff
@@ -69,19 +70,18 @@ def initialize(server, app_pool)
 
     private
 
-
     # Until a request is added via the `add` method this method will internally
     # loop, waiting on the `sockets` array objects. The only object in this
     # array at first is the `@ready` IO object, which is the read end of a pipe
     # connected to `@trigger` object. When `@trigger` is written to, then the loop
-    # will break on `IO.select` and return an array.
+    # will break on `select()` and return an array.
```

Comment:
I think all these `select()`'s are technically accurate but a bit confusing since `IO.select` and `NIO::Selector#select` are two different things. Can we be more accurate and say `NIO::Selector#select` instead?

Reply:
I don't see why not. Thanks for your review.